audit-resolve.json

```diff
@@ -4,8 +4,16 @@
       "decision": "ignore",
       "madeAt": 1626802771020,
       "expiresAt": 1640995200000
+    },
+    "1091239|opensearch-browser>isomorphic-fetch>node-fetch": {
+      "decision": "ignore",
+      "madeAt": 1692723196646
+    },
+    "1091791|opensearch-browser>isomorphic-fetch>node-fetch": {
+      "decision": "ignore",
+      "madeAt": 1692723202247
```

Comment:
These changes allow the CI workflow to ignore the security vulnerabilities being report for opensearch-browser/isomorphic-fetch/node-fetch.